fhd_core/gridding/visibility_grid.pro

```diff
@@ -94,6 +98,18 @@ psf_dim2=2*psf_dim
 psf_dim3=psf_dim*psf_dim
 bi_use_reduced=bi_use mod nbaselines
 
+if keyword_set(beam_per_baseline) then begin
+    uv_grid_phase_only=1
+    psf_intermediate_res=(Ceil(Sqrt(psf_resolution)/2)*2.)<psf_resolution
+    psf_image_dim=(*psf.image_info).psf_image_dim
+    image_bot=-Floor(psf_dim/2)*psf_intermediate_res+Floor(psf_image_dim/2)
+    image_top=(psf_dim*psf_resolution-1)-Floor(psf_dim/2)*psf_intermediate_res+Floor(psf_image_dim/2)
+
+    l_m_n, obs, psf, l_mode=l_mode, m_mode=m_mode, n_tracked=n_tracked
```

Comment:
As in `visibility_degrid`, I think it would be more clear with l_m_n written as a function:
`n_tracked = l_m_n(obs, psf, l_mode=l_mode, m_mode=m_mode)`

Reply:
Done